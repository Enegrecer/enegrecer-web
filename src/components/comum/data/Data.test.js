import React from 'react';
import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import Data from '../data';

describe('Data', () => {
  it('deve possuir estrutura definida ', () => {
    const onChange = jest.fn();
    const data = mount(<Data id={'13'} onChange={onChange} label="Data" divClasses={'estado'} />);
    const dataTextoJson = toJson(data);
    expect(dataTextoJson).toMatchSnapshot();
  });
});

