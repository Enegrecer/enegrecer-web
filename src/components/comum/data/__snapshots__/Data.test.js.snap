// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Data deve possuir estrutura definida  1`] = `
<Data
  divClasse=""
  divClasses="estado"
  estado=""
  id="13"
  label="Data"
  onChange={[Function]}
>
  <campoTexto
    divClasse=""
    id="13"
    inputClasse="datepicker"
    label="Data"
    maxLen={0}
    onChange={[Function]}
    placeholder="Data"
    type="text"
  >
    <div
      className=""
    >
      <input
        className="datepicker"
        id="13"
        maxLength={0}
        onChange={[Function]}
        placeholder="Data"
        type="text"
      />
      <label
        className="active"
        htmlFor="13"
      >
        Data
      </label>
    </div>
  </campoTexto>
</Data>
`;
